scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -1774,6 +1781,11 @@ public DtlsConnectorConfig build() {
 				throw new IllegalStateException("Supported cipher suites must be set either " +
 						"explicitly or implicitly by means of setting the identity or PSK store");
 			}
+			for (CipherSuite cipherSuite : config.supportedCipherSuites) {
+				if (!cipherSuite.isSupported()) {
+					throw new IllegalStateException("cipher-suites " + cipherSuite + " is not supported by JVM!");
+				}
+			}
```

Comment:
I suspect this test is not needed, because we already test it in setter and for default value `determineCipherSuitesFromConfig` should return only supported cipher ?

(but maybe it's clearer to keep it?)

Reply:
My concern was "should return". I just want to make sure, that the JVM support doesn't make more trouble then intended. But yes, though `determineCipherSuitesFromConfig` only return supported ones, I may also sharp the javadoc.

Reply:
Though currently the unit tests of 2.0.x fails without this PR, I would go to merge it. If wanted, we can remove the redundant check later.